api/update-docker.sh

```diff
@@ -8,4 +8,4 @@ fi
 
 echo "Running new process"
 
-docker run --rm -t -p 80:8080 jemisonf/msgr_api:latest &
\ No newline at end of file
+docker run --rm -t -p 80:8080 -v /home/ec2-user/cs340_final_project/conf/configuration.py:/app/configuration.py jemisonf/msgr_api:latest &
```

Comment:
Instead of using `&`, we could use Docker's detached mode by using `-d`. This will make sure stdout and stderr are being logged